ext/lib/mgmt/mcumgr/ext/base64/src/base64.c

```diff
@@ -0,0 +1,181 @@
+/*
+ * This file is based on roken from the FreeBSD source.  It has been modified
+ * to not use malloc() and instead expect static buffers, and tabs have been
+ * replaced with spaces.  Also, instead of strlen() on the resulting string,
+ * pointer arithmitic is done, as p represents the end of the buffer.
+ */
+
+/*
+ * Copyright (c) 1995-2001 Kungliga Tekniska Högskolan
+ * (Royal Institute of Technology, Stockholm, Sweden).
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
```

Comment:
looks like this is not pure APL, we have some BSD code in this import, @carles what do we do about this?

Reply:
mbedtls has bas64 build-in and is on apache 2.0, we can us it instead of this.

Reply:
Agree with @nvlsianpu. @ccollins476ad would it be hard to use the mbedtls one instead? It's also quite ugly that we have an `ext` folder nested inside `ext/`.

Reply:
I have removed this base64 library (and its parent `ext` directory).  I included a few more details in this comment: https://github.com/zephyrproject-rtos/zephyr/pull/6068#discussion_r167756541.